inc/thorvg.h

```diff
@@ -143,8 +143,9 @@ enum class TVG_EXPORT CompositeMethod
 {
     None = 0,     ///< No composition is applied.
     ClipPath,     ///< The intersection of the source and the target is determined and only the resulting pixels from the source are rendered.
-    AlphaMask,    ///< The pixels of the source and the target are alpha blended. As a result, only the part of the source, which intersects with the target is visible.
-    InvAlphaMask  ///< The pixels of the source and the complement to the target's pixels are alpha blended. As a result, only the part of the source which is not covered by the target is visible.
+    AlphaMask,    ///< The pixels of the source and the target are alpha blended. As a result, only the part of the source, which alpha intersects with the target is visible.
+    InvAlphaMask, ///< The pixels of the source and the complement to the target's pixels are alpha blended. As a result, only the part of the source which alpha is not covered by the target is visible.
+    ColorMask     ///< The source pixels are converted to the grayscale and alpha blended with the target. As a result, only the part of the source, which intersects with the target is visible.
```

Comment:
Please add @BETA_API tag to ColorMask

Reply:
Suggestion: ColorMask =>  LumaMask
https://en.wikipedia.org/wiki/Luma_(video)

Reply:
Renamed to LumaMask